AmericanMedicalAssociation/ama-style-guide-2
# EWL-8394 Composer V2 Upgrade

## Ticket(s)

**Github Issue**
N/A

**Jira Ticket**
- [EWL-8394: Composer V2 Upgrade](https://issues.ama-assn.org/browse/EWL-8394)

## Description
Changes all references to current repo account name "AmericanMedicalAssociation" to composer enforced "american-medical-association". Accompanies D8 PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/2398.


## To Test
- See [D8 PR](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2398) for testing instructions and details regarding this change.

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
